app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :set_commentable, only: %i[create destroy]
+
+  def create
+    @comment = current_user.comments.build(comment_params)
+    @comment.commentable = @commentable
+    @comment.save
```

Comment:
```suggestion
    @comment.save!
```

万一保存に失敗したときは例外を出して気付きやすくした方が良いです

Reply:
承知しました！